browsers/safari.json

```diff
@@ -173,6 +173,19 @@
           "status": "current",
           "engine": "WebKit",
           "engine_version": "610.1.28"
+        },
+        "14.0.2": {
```

Comment:
See https://github.com/mdn/browser-compat-data/pull/8838 for a previous discussion on how to handle `MediaRecorder` and iOS 14.3. The conclusion then was to treat it as a minor version and live with a little bit of inaccuracy in the data.

But if the Safari teams considers these versions significant and would like to add them, that's a decision that should be revisited. cc @ddbeck @Elchi3

Reply:
We are opting for fudging this data and using "14" in cases where `14.0.2` or `safari_ios` shows `14.3`.